pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+import Head from 'next/head'
+import styled from 'styled-components'
+import { Content } from '../components/Content/Content'
+import { H1 } from '../components/Title/Title'
+import { Footer } from '../components/Footer/Footer'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { Gallery } from '../components/Gallery/Gallery'
+import { InfoBox } from '../components/InfoBox/InfoBox'
+
+const Section = styled.div`
+
+  margin-top: 3rem;
+  margin-bottom: 1rem;
+
+  h2 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.8rem;
+  }
+
+  h3 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.rem;
+  }
+
+  h4 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1rem;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  svg {
+    vertical-align: middle;
+    margin-top: -3px;
+    opacity: 0.7;
+    margin-right: 0.3rem;
+  }
+`
+const Device = styled.div`
+
+  display: flex;
+  margin-top: 1rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: rgba(108, 108, 108, 0.09);
+  padding: 1rem;
+  padding: 1.5rem;
+  border-radius: 0.5rem;
+  box-sizing: border-box;
+
+  p, h3 {
+    width: 100%;
+  }
+
+  div {
+    width: 50%;
+    margin-top: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: stretch;
+    flex-wrap: wrap;
+  }
+
+  div > ul {
+    width: 100%;
+    margin-top: -1rem;
+  }
+
+  div > ul > li {
+    height: 70%;
+  }
+`;
+
+const DeviceAmount = styled.div`
+  text-transform: uppercase;
+  color: rgba(255,255,255,0.6);
+  width: auto !important;
+  text-align: left;
+  margin: 0 0 1rem 0 !important;
+`
+const SpecList = styled.div`
+
+  display: flex;
+  margin-top: 1rem;
+  flex-direction: column;
+  min-width: 50%
+  height: 
+  justify-content: flex-start;
+
+  p {
+    width: 100%;
+    margin-top: 0.2rem;
+  }
+
+  @media(max-width: 1090px) {
+    width: 100% !important;
+  }
+`
+
+const ResponsiveSplitFlex = styled.div`
+
+  display: flex;
+  width: 100% !important;
+  flex-wrap: nowrap !important;
+
+  div > div {
+    height: 80%;
+    width: 100%;
+  }
+
+
+  @media(max-width: 1090px) {
+    flex-direction: column-reverse;
+    align-items: flex-start;
+
+    div:nth-child(2) {
+      flex-wrap: nowrap;
+      width: 100%;
+      height: 20rem;
+
+      ul {
+        margin-top: 1rem;
+        margin-left: 1rem;
+      }
+
+      ul > li {
+        height: 3rem;
+        margin-right: 100%;
+        margin-bottom: 0.5rem;
+      }
+    }
+  }
+
+  @media(max-width: 776px) {
+
+    div:nth-child(2) {
+      width: 100%;
+      height: auto;
+      flex-wrap: wrap;
+      margin-top: unset;
+      height: 25rem !important;
+
+      ul {
+        margin-top: 1rem;
```

Comment:
The `margin-top: 1rem;` and `margin-bottom: 1rem;` can be combined into `margin-block: 1rem;`

Reply:
Fixed in #[e4125c5](https://github.com/Testausserveri/testausserveri.fi/pull/99/commits/e4125c56cf8cab9d9ec0c55e15c9f1b8876d5c51)